modin/experimental/xgboost/utils.py

```diff
@@ -0,0 +1,52 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import logging
+import xgboost as xgb
+
+LOGGER = logging.getLogger("[modin.xgboost]")
+
+
+def start_rabit_tracker(num_workers: int, host_ip):
```

Comment:
why it is not a context manager for the tracker?.. its `__enter__()` could return the tracker, not the context manager, making stopping a tracker automated...

Reply:
Context manager was added instead of `start/stop..` separate functions.